Setting Up Neolink For Use With Blue Iris.md

```diff
@@ -0,0 +1,125 @@
+1. In the Reolink PC app, login to your camera.
+2. Click "Device Settings" (the gear) -> "Network General."
+3. Change "Connection Type" from "DHCP" to "Static."
+4. Enter a static IP address compatible with your network (i.e. `192.168.1.15`)
+
+_You will have to reconnect to the camera once you have changed the IP address_
+
+### 3. Set the camera's time to your local network time.
+_If the camera's time is not set, Neolink will recursively "time out" every one second and will not stream video._
+1. In the Reolink PC app, login to your camera.
+2. Click "Device Settings" -> "System General" -> "Synchronize Local Time."
+3. Click "Ok."
+
+### 4. Disable Auto Reboot
+_When a camera reboots, it loses its date and time settings, causing Neolink to time out._
+1. In the Reolink PC app, login to your camera.
+2. Click "Device Settings" -> "Maintenance."
+3. Uncheck "Enable Auto Reboot."
+
+### 5. Set a Password
+_It's recommended that you set a password for each of your cameras. If you want to use the Reolink Mobile App, it makes you set a password for each camera anyway._
+1. In the Reolink PC app, login to your camera.
+2. Click "Device Settings" -> "Manage User."
+3. Click "Modify Password."
+
+**Now you've set up your cameras!**
+
+## Setting Up Neolink
+### 1. Installation
+Refer to [Neolink's README](https://github.com/thirtythreeforty/neolink/blob/master/README.md) for instructions on installing.
+### 2. Create your config file.
+The config file tells Neolink how to connect to your camera and serve the video streams.
+Note: the config file's file extension _**must**_ be `.toml` to work properly.
+1. Create a simple text file (i.g. `config.toml`) in the same directory you have unpacked Neolink with the following format:
+
+        bind = "0.0.0.0"
+        
+        [[cameras]]
+        name = "cameraname"
+        username = "admin"
+        password = "password"
+        address = "192.168.1.10:9000"
+        stream = "both"
+    
+2. Change `cameraname` to a legible, phonetic name that describes your camera. Leave the quotes around the name.
+3. The default username is `admin`; leave this unless you've created another user.
+4. Replace `password` with the password you set on the camera. If you chose to not use a password, remove this line from the config file. Again, leave the quotes.
+5. Replace `192.168.1.10:9000` with the IP address you set for your camera. 
+    Note: The port, `:9000`, should remain at the end of your IP address. This is the proprietary "media port" that Reolink uses.
+6. The `stream` line allows you to choose which stream type to use. Neolink supports streaming two streams, the main-stream, and the sub-stream. It can stream either one, or both. If you wish to stream both streams, leave this line as is. If you wish to stream _only_ the main-stream, change `both` to `mainStream`. If you wish to stream only the sub-stream, change `both` to `subStream`.
```

Comment:
Remove this line from the easy setup, since most users want substream. Maybe we can add a "configuration manual" that exhaustively documents all the config options.

Reply:
What about D/B400 cams which only support using one stream?

Reply:
Good point. Maybe add a section about 400 cams?

Reply:
I'll add this tonight!